### PR TITLE
Add a confidence metric to most trusted rules

### PR DESCRIPTION
**Pull request by Taiki-San**
### Checklist
- [ ] Did you added the `test_vectors` for your rules
- [ ] Did you tested it on your local machine first
- [ ] Did you tested it on staging

### CODE REVIEW

File: rules/recommended/sqr-000-015.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name: 'SSRF: Detect SSRF attempt using non HTTP protocol'
 tags:
   type: ssrf
   category: attack_attempt
+  confidence: "0"
```
Comment (netcode) [inline rules/recommended/sqr-000-015.yaml:6]: I would say this is rarely a legit use, especially for rare protocols like gopher and dict.

Comment (Taiki-San) [reply rules/recommended/sqr-000-015.yaml:6]: My biggest problem with this rule was `file://` which is used by a few service to refer to local files of Android applications :/

File: rules/recommended/crs-920-260.yaml [hunk 2]
```diff
@@ -4,6 +4,7 @@ tags:
   type: http_protocol_violation
   crs_id: '920260'
   category: attack_attempt
+  confidence: "1"
```
[the remark this answers is not in the document]
Comment (Taiki-San) [reply rules/recommended/crs-920-260.yaml:7]: That's a pretty specialised subset of Unicode code points though. I had a look again and most triggers appear valid to my eye

File: rules/recommended/sqr-000-015.yaml (hunk 1 above)
Comment (netcode) [reply rules/recommended/sqr-000-015.yaml:6]: why web services calling local files in an android application? It seems so odd to me.

File: rules/recommended/crs-920-260.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (netcode) [reply rules/recommended/crs-920-260.yaml:7]: I saw some false positive. 
But anyway this rule was suppose to fight abuse of using full-width/half-width unicode bypass. Which if im not mistaken a very old bypass which is not surfaced in the last 15 year

**Review by netcode — APPROVED** [review]

**Pull request merged**